frontend/src/components/PostCard.jsx

```
@@ -220,10 +221,33 @@ const PostCard = ({
-  const formatTime = (timeString) =>
-    timeString
-      ? new Date(timeString).toLocaleTimeString([], { hour: "2-digit", minute: "2-digit" })
-      : "Flexible";
+  // Fix for time zone issue: Update this in PostCard.jsx
+const formatTime = (timeString) => {
+  if (!timeString) return "Flexible";
+  
+  try {
+    // Extract just the time portion from the ISO string
+    const timeMatch = timeString.match(/T(\d{2}:\d{2})/);
+    if (timeMatch && timeMatch[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)